website/src/content/docs/docs/release-notes/release-2025-03-18.mdx

```diff
@@ -45,19 +45,19 @@ TypeSpec core contains Breaking Changes
 
 ### @azure-tools/typespec-client-generator-core
 
-- [#2298](https://github.com/Azure/typespec-azure/pull/2298) Add `pageItemsSegments` for `SdkPagingServiceMetadata` to indicate how to get page items from response.
-- [#2390](https://github.com/Azure/typespec-azure/pull/2390) Specific scope decorator should always override all-scopes decorator
+- [#2298](https://github.com/Azure/typespec-azure/pull/2298) Add `pageItemsSegments` for `SdkPagingServiceMetadata` to indicate how to get page items from response.
```

Comment:
@tadelesh can you also apply the same fix to the change file so that will apply to the changelog.md when we release

Reply:
done in this pr: https://github.com/Azure/typespec-azure/pull/2269.